layer/projects/project_runner.py

```diff
@@ -166,23 +166,25 @@ def run(self, debug: bool = False, printer: Callable[[str], Any] = print) -> Run
                     )
         return run
 
-    async def _upload_tar_packages(self, client: LayerClient) -> None:
+    async def _upload_executable_packages(self, client: LayerClient) -> None:
         async with aiohttp.ClientSession(raise_for_status=True) as session:
             upload_tasks = [
-                self._upload_tar_package(client, definition, session)
+                self._upload_executable_package(client, definition, session)
                 for definition in self.definitions
             ]
             await asyncio.gather(*upload_tasks)
 
-    async def _upload_tar_package(
+    async def _upload_executable_package(
         self,
         client: LayerClient,
         function: FunctionDefinition,
         session: aiohttp.ClientSession,
     ) -> None:
-        with open(function.tar_path, "rb") as package_file:
+        package_path = function.package()
```

Comment:
We are already doing the packaging here https://github.com/layerai/sdk/blob/fb41d2d8e010ae29d1c5603df076b9fc0b6ca073/layer/projects/project_runner.py#L117 . I think one of them should be removed.

Reply:
Good catch, removed the packaging from the upload in https://github.com/layerai/sdk/pull/197/commits/b1bce8255bd5f81167ed0e689e94707c9359ceb9.